### PR TITLE
Move db validation to switch case

### PR DESCRIPTION
**Pull request by noah8713**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -189,6 +189,15 @@ type ovndb struct {
...
 	// db string should strictly be OVN_Northbound or OVN_Southbound
+	switch db {
+	case DBNB, DBSB:
+		break
+	case "":
+		db = DBNB
+	default:
+		return nil, fmt.Errorf("Valid db names are: %s and %s", DBNB, DBSB)
+	}
+
```
Comment (hzhou8) [inline client.go]: The change doesn't seem necessary. Meanwhile, L201-206, the old validation, were not removed.

Comment (vtolstov) [reply client.go]: why? its more readable now

Comment (noah8713) [reply client.go]: @hzhou8 :Yes removed l201-206. My bad in testing!

Comment (hzhou8) [reply client.go]: > why? its more readable now

Well, it doesn't look more (or less) readable to me, but it is good if at least it is more readable to you :)

**Comment by noah8713** [comment]
Thanks for the review guys!

**Pull request merged**